libraries/Adapters/Microsoft.Bot.Builder.Adapters.WeChat/WeChatClient.cs

```diff
@@ -0,0 +1,726 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Adapters.WeChat.Schema;
+using Microsoft.Bot.Builder.Adapters.WeChat.Schema.JsonResult;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Adapters.WeChat
+{
+    public class WeChatClient
+    {
+        private static readonly string APIHost = "https://api.weixin.qq.com";
+        private static readonly HttpClient HttpClient = new HttpClient();
```

Comment:
yes, HttpClient is disposable, but the [best practice for HttpClient](https://docs.microsoft.com/en-us/aspnet/web-api/overview/advanced/calling-a-web-api-from-a-net-client) says you should create one instance and reuse it as much as possible. Therefore, here I'll never dispose of it simply because I will be needing it all the time.